### PR TITLE
Upgrade sentry to 23.8.0

### PR DESCRIPTION
**Pull request by adonskoy**
Sentry is updated to 23.8.0 and new issue-platform components are added

### CODE REVIEW

File: sentry/templates/configmap-sentry.yaml [hunk 1]
```diff
@@ -448,6 +448,7 @@ data:
                 "organizations:profiling-memory-chart",
                 {{ end -}}
 
+                "organizations:issue-platform",
```
Comment (TartanLeGrand) [inline sentry/templates/configmap-sentry.yaml]: Move this before if for have all "organizations" together.

**Review by TartanLeGrand — APPROVED** [review]

File: sentry/templates/deployment-sentry-ingest-occurrences.yaml [hunk 2]
```diff
@@ -0,0 +1,144 @@
+{{- if .Values.sentry.features.enableProfiling }}
```
Comment (Mokto) [inline sentry/templates/deployment-sentry-ingest-occurrences.yaml]: Just to make sure, ingest-occurrences is only needed for profiling?

Comment (adonskoy) [reply sentry/templates/deployment-sentry-ingest-occurrences.yaml]: No, this condition should not be here. Thanks for noticing!

**Comment by adonskoy** [comment]
@Mokto, is there anything else that needs to be done before the merge?

**Review by Mokto — APPROVED** [review]

**Comment by Mokto** [comment]
Thanks!

**Pull request merged**